### PR TITLE
Added new stylesheet and various animations

### PR DESCRIPTION
**Pull request by himanshubobade**
I have added a new style sheet with my own color preferences with amazing 3D card styling and a simple home page. I hope u like it. Also i have added screenshots for the same.
The Happy Chameleon - Google Chrome 23-10-2020 12 16 41 AM
The Happy Chameleon - Google Chrome 23-10-2020 12 16 54 AM
The Happy Chameleon - Google Chrome 23-10-2020 12 17 04 AM

### CODE REVIEW

File: stylesheets/style4.css [hunk 1]
```diff
@@ -0,0 +1,365 @@
...
+    margin: 0 40%;
+}
+
+.footerWrap .col2 svg,
+.footerWrap .col2 svg,
+.footerWrap .col2 svg,
+.footerWrap .col2 svg{
+    height: 30px;
+    fill: var(--font);
+}*/
```
Comment (faisalAkhtar) [inline stylesheets/style4.css]: These are not needed, you can remove them

**Review by faisalAkhtar — CHANGES_REQUESTED** [review]
Please make the stylesheet responsive.

File: stylesheets/style4.css (hunk 1 above)
Comment (faisalAkhtar) [reply stylesheets/style4.css]: All in all, this stylesheet @himanshubobade is fantastic :100: 

Please make it responsive and then we will get it merged.

Comment (himanshubobade) [reply stylesheets/style4.css]: @faisalAkhtar yes will do soon

**Pull request closed without merging**